test/snapper_dbus_test.rb

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env rspec
+# coding: utf-8
+
+ENV["Y2DIR"] = File.expand_path("../../src", __FILE__)
+
+require "yast"
+
+Yast.import "Snapper"
+
+describe "SnapperDbusClass" do
+
+  describe "#escape" do
+
+    it "call with nothing special" do
```

Comment:
this is not usage statement. better something like `it "do not change simple string"`